### PR TITLE
Nikita 861mkrxmn

### PR DESCRIPTION
**Pull request by nikitunya**
https://app.clickup.com/t/861mkrxmn

Usability: Add "New deck" button in the navbar and add Sidebar

### CODE REVIEW

File: flashcards/app/Http/Resources/Deck/DeckResource.php
Comment (Siamino) [inline flashcards/app/Http/Resources/Deck/DeckResource.php:1]: Why are these changed needed for the backend?

**Review by Siamino — CHANGES_REQUESTED** [review]

File: flashcards/app/Http/Resources/Deck/DeckResource.php
Comment (aivaraskriksciunas) [reply flashcards/app/Http/Resources/Deck/DeckResource.php:1]: @nikitunya Please answer Luka's question

**Comment by Siamino** [comment]
merge request into master? are you a junior dev?

**Review by aivaraskriksciunas — APPROVED** [review]
